ush/ioda/bufr2ioda/bufr2ioda_acft_profiles_prepbufr.py

```diff
@@ -99,6 +99,10 @@ def bufr_to_ioda(config, logger):
     bufrfile = f"{cycle_type}.t{hh}z.{data_format}.acft_profiles"
     DATA_PATH = os.path.join(dump_dir, f"{cycle_type}.{yyyymmdd}", str(hh),
                              'atmos', bufrfile)
+    if not os.path.isfile(DATA_PATH):
+        logger.info(f"DATA_PATH {DATA_PATH} does not exist")
+        return
```

Comment:
Should this result in an `exception`?  Should it be a `warning`?

Reply:
From time to time we have missing or zero length dump files.  An `exception` causes the entire prepatmiodaobs job to fail.  Hence the above to allow prepatmiodaobs to finish with other dump files it successfully processes.  This is not intended to be a long term solution.  It was an effort to allow the February 2024 JEDI ATM parallel to run.  

We could query expected dump files and only run the converters on non-zero length dump files.  Even if this is done, we should harden our converters to handle a wide range of dump file possibilities.  

My preference is to let the DA team working on bufr2ioda converters discuss and decide how to handle dump file issues.   Once agreement is reached a consistent approach can be used in all converters.